tests/phpunit/admin/class-wc-stripe-admin-notices-test.php

```
@@ -797,6 +803,87 @@ public function provide_test_subscription_check_detachment() {
+	/**
+	 * Test that a subscription whose ID is in the dismissed array does not show the detachment notice.
+	 *
+	 * @return void
+	 */
+	public function test_subscription_detached_notice_hidden_for_dismissed_subscription() {
+		global $theorder;
+		$original_order = $theorder;
+
+		update_option( 'wc_stripe_dismissed_subscription_detached_notice', [ 123 ] );
+
+		$subscription = new WC_Subscription();
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '817', column '23'). (undefined)

(MissingImport)